libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/speedlimit/DropInSpeedUIBinder.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.dropin.component.speedlimit
+
+import android.transition.Scene
+import android.transition.TransitionManager
+import android.view.ViewGroup
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.binder.UIBinder
+import com.mapbox.navigation.dropin.databinding.DropInSpeedLimitViewBinding
+import kotlinx.coroutines.InternalCoroutinesApi
+
+@OptIn(InternalCoroutinesApi::class, ExperimentalPreviewMapboxNavigationAPI::class)
+class DropInSpeedUIBinder : UIBinder {
```

Comment:
Can we remove the term `DropIn` and just call it `SpeedLimitViewBinder` just like we are doing it with the screens? @kmadsen

Reply:
👍